src/Symfony/Component/Ldap/Adapter/ExtLdap/EntryManager.php

```diff
@@ -67,6 +67,47 @@ public function remove(Entry $entry)
         }
     }
 
+    /**
+     * Adds values to an entry's multi-valued attribute from the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $attribute
+     * @param array  $values
```

Comment:
The 3 `@param` entries can be removed (we don't add those if they don't add something that is not part of the PHP method signature already).